src/Launcher/Launcher.jl

```diff
@@ -648,7 +648,9 @@ function launch!(
         J = Vector{Vector{Float64}}([[x...] for x in J])
 
         # normalize couplings and level repulsion
-        J, A = normalize([J, [[A]]])
+        JAtemp = normalize([J, [[A]]])
+        J = JAtemp[1]
+        A = JAtemp[2][1][1]
```

Comment:
Ok, this fixes my previous concern. Can we make the `=` signs align?

Reply:
I really dislike adding tons of whitespace for hardly any more readybility (in my opinion in this case it would read worse... But for the sake of consistency...

Reply:
I agree, here it does not really add anything, but, yeah, consistency ...